### PR TITLE
WIP: Dbusel/feature ewc 221

### PR DESCRIPTION
**Pull request by dbusel**

### CODE REVIEW

File: src/schemas/microservice.js [hunk 1]
```diff
@@ -45,11 +47,12 @@ const microserviceUpdate = {
     "volumeMappings": {
       "type": "array",
       "items": {"$ref": "volumeMappings"}},
-    "ports": {"$ref": "/ports"},
+    "ports": {
+      "type": "array",
+      "items": {"$ref": "ports"}},
```
Comment (epankou) [inline src/schemas/microservice.js]: are you sure that here should be "$ref": "ports" ?
Subschema ports has id "/ports" right now.

File: src/schemas/microservice.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ const microserviceCreate = {
...
```
Comment (epankou) [inline src/schemas/microservice.js]: are you sure that here should be "$ref": "ports" ?
Subschema ports has id "/ports" right now.

File: src/schemas/microservice.js (hunk 1 above)
Comment (dbusel) [reply src/schemas/microservice.js]: done

File: src/schemas/microservice.js (hunk 2 above)
Comment (dbusel) [reply src/schemas/microservice.js]: done

File: src/schemas/microservice.js [hunk 3]
```diff
@@ -58,7 +61,8 @@ const ports = {
   "type": "object",
   "properties": {
     "internal": {"type": "integer"},
-    "external": {"type": "integer"}
+    "external": {"type": "integer"},
+    "publicMode": {"enum": [false]}
```
Comment (epankou) [inline src/schemas/microservice.js]: looks weird, what does it mean?

Comment (dbusel) [reply src/schemas/microservice.js]: it's hardcode for now

**Pull request merged**